### ISSUE TITLE
Implement metadata model/data storage strategy for Imaging Mass Cytometry

### ISSUE THREAD

**Issue by Bankso** [mc2-center/data-models#73]
Imaging Mass Cytometry data is being generated by grants supported by the MC2 Center. Recent interactions with a community member are likely to result in the submission of this data to Synapse. To support storage and sharing of IMC data, we should do the following (not necessarily exhaustive or in order):

- identify existing models and storage strategies for IMC data
- refine models and storage strategies to align with existing MC2 models and approaches
- implement model and storage strategies
- work with contributors to test model and storage strategies on data
- record information on model, storage strategy, annotation workflow, etc. in an SOP document
- determine if there is a CRDC node that will take this data or if Synapse should be considered the destination repo

Initial work on the steps above can be found in this folder, [IMC_model_development](https://drive.google.com/drive/folders/17K152Xkv2i72wt25pAlIMBqBna2CmIlq)

This ticket will be updated with timelines once more information has been received from the potential contributor.

**Comment by aclayton555** [comment]
If picking this up, consider building off updated imaging approach for HTAN2 - see RFC here: https://docs.google.com/document/d/1mVRXyqSoosyZ4vCGDU86nGGyL_EWR75fEbSXWiCDAIA/edit?usp=sharing

**Comment by Bankso** [comment]
I don't think we should plan to add an Imaging Mass Cytometry model until we've defined models for it's "component" assays (mass spectrometry/cytometry, flow cytometry). 

Using these models, augmented with attributes from Imaging, we can use a similar sort of approach as we did with the GeoMx model (capturing spatial information separately) to synthesize an Imaging Mass Cytometry model (I described this a bit in the confluence page I wrote up for #217 )
